src/scenic/core/utils.py

```diff
@@ -216,6 +217,95 @@ def unifyMesh(mesh, verbose=False):
         return mesh
 
 
+def repairMesh(mesh, pitch=(1 / 2) ** 6, verbose=True):
```

Comment:
If we're going to recommend ordinary users use this function to fix their meshes, we'd better document the parameters and return value.